docs/docs/0.3.0/policies/README.md

```diff
@@ -206,6 +209,14 @@ spec:
         service: redis
         version: '2.0'
 ```
+You also need to add labels to your services and use that label as a matching tag. Above, you  see that we utilize the `version` tag to help with canary deployment. Another common use-case is to add a `env` tag as you separate testing, staging, and production environments' services.
+
+#### Resolving Collision in Traffic Routes
+In the event where two traffic route many have colliding configurations, these are the expected behaviors:
+* Identical Selectors: the policy will be prioritized alphabetically by name
+* More tags: The policy with more matching tags will be prioritized first
```

Comment:
Let's give an example, e.g. `policy that matches "source" dataplanes by 2 tags, e.g. "service" and "env", will win over a policy that matches "source" dataplanes only by 1 tag, e.g. "service"`